BUILD.windows.md

```diff
@@ -1,43 +1,67 @@
-Build instructions for Windows 10
-=================================
+# Build instructions for Windows 10/11 Pro
 
-Environment Setup
------------------
+## Environment Setup
 
 ### Chocolatey
 
-Install chocolatey, a package manager, to download the rest of the dependencies: <https://chocolatey.org/>
+Install chocolatey, a package manager, to download the rest of the dependencies: <https://chocolatey.org/>.
 
-Press Windows Key+X and Run Windows PowerShell(Admin) then follow the chocolatey instructions to "Install with
-Powershell.exe"
+Press Windows Key+X and Run Windows PowerShell(Admin) then follow the chocolatey instructions to
+"Install with Powershell.exe".
 
-After chocolatey is installed you can now install the rest of the dependencies:
+### Dependencies
 
-    choco install visualstudio2019buildtools visualstudio2019-workload-vctools cmake ninja cmder qemu-img openssl -yfd
+After chocolatey is installed you can now install the rest of the dependencies from the
+Powershell(Admin). To get the best results, in the following order:
 
-You may have to disable Windows Defender Real-time protection if you want the packages to install quicker. Search for
-Windows Defender Security Center, go to Virus & threat protection, then Virus and thread protection settings, disable
-Real-time protection.
+```[pwsh]
+choco install cmake ninja qemu openssl git wget unzip -yfd
+```
+
+```[pwsh]
+choco install visualstudio2019buildtools visualstudio2019-workload-vctools -yfd
+```
+
+You may have to disable Windows Defender Real-time protection if you want the packages to install
+quicker. Search for Windows Defender Security Center, go to Virus & threat protection, then Virus
+and thread protection settings, disable Real-time protection.
```

Comment:
Being an open-source repo with community influence, I think we should avoid this altogether and never recommend turning off security features. I would reserve this info only for developer documentation or on a case-by-case basis if a contributor is being bothered by the setup process.